Userland/Libraries/LibGfx/QOICommon.h

```diff
@@ -18,7 +18,8 @@ namespace Gfx {
 // Implementation details for the "Quite OK Image" format (v1.0).
 // https://qoiformat.org/qoi-specification.pdf
 
-static constexpr auto QOI_MAGIC = "qoif"sv;
+static constexpr u8 QOI_MAGIC_SIZE = 4;
+static constexpr u8 QOI_MAGIC[QOI_MAGIC_SIZE] = { 'q', 'o', 'i', 'f' };
```

Comment:
why?

Reply:
(this could just be a `Span<u8 const>` to make the comparison below even easier)